### ISSUE TITLE
Seared Duct not actually working (but seems to)

### ISSUE THREAD

**Issue by eslachance** [Toops/TinkersSteelworks#8]
There seems to be a bit of wonkiness going on with using a Seared Duct to feed Ex Nihilo's special "Ore Dust" blocks into the High Oven. If I just throw a stack of the stuff in the Controller, it works fine. More than fine: Getting tripple ore is almost orgasmic. Next time I start a world, screw pulverizers, amirite?

However, if I start with a completely empty controller (except for fuel and a 3000 degree temperature, wooh!), and I place a single Ore Dust in the Seared Duct, set the output to the "ingot" icon (I assume this is the right setting, you need a bit more documentation in your book to explain the icons and all the slots, man!) and give it redstone from a lever, it doesn't work. That is to say, the dust doesn't get deposited in the controller, even though visually it seems to disappear from the duct's interface. But if I exit the interface and go back in it, I can see the dust for a fraction of a second before it disappears again. Troubleshooting, I noticed that if I switch the icon at the bottom, then the dust (or any amount of dust I throw in that disappears initially) just... pops back into the slots, like it never left.

Things can get even wonkier though. If I already have, say, a full stack of dust in the controller getting melted, and I place some more dusts in the duct (redstone on, ingot icon), then I get the following:
- While looking at the duct GUI, the number of dusts in the first slot goes down, fast. Way too fast, much faster than it should. But, if I exit the GUI and re-open it, magically I still have 64 dusts in the first slot and all other stacks are untouched. >.<
- If I look at the Controller GUI it's even weirder: Say I have a full stack, and I put a stack in the Duct GUI, the numbers in the stack go very wonky: When the first ingot is burned, the stack size goes down to 63 and back up to 64... alright... second burn, it goes down to 62, then goes to 63 and 64. Third burn, goes to 61, 62, 63, 64.... you get the picture I guess. Through all this I'm still just getting my 3 ingots of molten metal in the tank, that just works.
- Extra weird: at one point it seems to go back to "normal": around 51 or 52 left in the stack, it stops building back up to 64, it just removes one per burn and we're good... until I look at the Duct GUI, where the stack size goes down (not really, again exit and go back in the Duct GUI and I still have my full stack). And then I go back to the controller and for another 10 burns or so, it builds back to 64 on each burn.

Basically, the duct just... doesn't work. It _seems_ to work, in the sense that it wants to add blocks to my stack in the controller and remove them from the duct, but it just kinda fakes it (like a bored wife would, y'know?). (Edit: I put the duct in the middle, on one of the sides like I did the controller and drain. Pumping items into the duct works from a hopper.)

BTW, using 0.0.3.1 in the Agrarian Skies mod pack/skyblock map.

**Comment by wisthy** [comment]
I have the same issue.
- Agrarian Skies modpack 2.0.3
- TConstruct_mc1.6.4_1.5.5d3
- TSeelworks-mc1.6.4_0.0.3.1

If I play with the interface (like switching the config of the duct from ingot slot to coal slot), it start pull items from it and when I watch the controller, I see item comings. If I leave the controller and open it again, it start again to pull from zero.

I tried without any itemduct, chest, hopper, just putting the item in it manually. I tried with itemduct and with hopper too. Same behavior each time.

Same behavior with or without redstone signal sent to the duct.

**Comment by jnachtigal** [comment]
I'm having  the same issue in Agrarian Skies 2.1.1 and the Duct will not feed in fuel even when set correctly.

**Comment by hubertnnn** [comment]
Same issue here (Agrarian Skies 2.1.2).
I found that there is slug appearing in duct set as ore input (the ingot icon). Also coal is bugging same way eslachance said in her post above. 

It looks like the mode is set only client side (coal movement is updating), but on server everything is set as output (thats why slag appeared in one of my ducts). 

Thats only a 
hypothesis, but it may help finding and fixing source of problem.

**Comment by wisthy** [comment]
Just tested with the new 0.0.4.1 version of TSteelworks (manually replaced in Agrarian Skies 2.1.2) and it works fine now.

Just have to wait (made a request?) for this updated mod to be included in the next version of Agrarian Skies

**Comment by Duckle29** [comment]
Nice to see I'm not the only one. Will talk to my server admin, and see about having our mod updated :)

**Comment by Toops** [comment]
Well, it seems this issue has been wrapped up nicely, so it's closing time.

Thanks again for your amazingly detailed report, eslachance!
